src/rgw/rgw_file.h

```diff
@@ -91,7 +91,7 @@ namespace rgw {
    */
   struct fh_key
   {
-    rgw_fh_hk fh_hk;
+    rgw_fh_hk fh_hk {0};
```

Comment:
I don't see a constructor in `rgw_fh_hk` that accepts an integer to do this change.

Otherwise, it should be like this: http://en.cppreference.com/w/cpp/language/value_initialization

Reply:
i think `rgw_fh_hk fh_hk {};` is good here

Reply:
@cbodley Yes, I meant the same. Thanks.